cypress/support/pages/signIn.pageObject.js

```diff
@@ -3,16 +3,19 @@ import PageObject from '../PageObject';
 class SignInPageObject extends PageObject {
   url = '/user/login';
 
-  get emailField() {
-    return cy.getByDataCy('email-sign-in');
+  emailField(email) {
```

Comment:
Function names should typically be verbs or verb phrases. Consider renaming 'emailField' to 'enterEmail' or something similar that reflects the function's action.